integration/tests/cook/util.py

```diff
@@ -1732,6 +1732,39 @@ def rebalancer_interval_seconds():
     return interval_seconds
 
 
+__auto_sequence_lock = threading.Lock()
+__auto_sequence_counter = 0
+
+
+def send_progress_update(cook_url, instance, assert_response=True, allow_redirects=True,
+                         sequence=None, percent=None, message=None, json=None):
+    """Submit a job instance progress update via the rest api"""
+    global __auto_sequence_counter
```

Comment:
Tests should explicitly provide their sequence number instead of us incrementing a global variable. In such a scenario `None` can be used to signal that we do not wish to send the sequence in the json payload like `percent` and `message`..

Reply:
Done